yb-voyager/cmd/migration_complexity.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"encoding/csv"
+	"fmt"
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/samber/lo"
+	log "github.com/sirupsen/logrus"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/constants"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+	"golang.org/x/exp/slices"
+)
+
+const NOT_AVAILABLE = "NOT AVAILABLE"
+
+var (
+	LEVEL_1_MEDIUM_THRESHOLD = 20
+	LEVEL_1_HIGH_THRESHOLD   = math.MaxInt32
+	LEVEL_2_MEDIUM_THRESHOLD = 10
```

Comment:
not blocker: let's discuss these numbers once with everyone

Reply:
yeah sure, its already in the Functional Spec doc.